src/components/types.ts

```diff
@@ -226,7 +227,11 @@ export interface ChartTooltipProps {
 }
 
 /** The ChartSimpleProps unifying all the possible types fot the simple chart props. */
-export type ChartSimpleProps = ChartBarProps | ChartLineProps | ChartAreaProps | ChartScatterProps;
+export type ChartSimpleProps = ChartBarProps | ChartLineProps | ChartAreaProps | ChartScatterPropsExt;
+
+export interface ChartScatterPropsExt extends  Omit<ChartScatterProps, 'dataComponent'> {
```

Comment:
Same as above, could you please move this before the `ChartSimpleProps` line? :)